rotkehlchen/data_migrations/manager.py

```diff
@@ -45,15 +45,16 @@ def __init__(self, rotki: 'Rotkehlchen'):
 
     def maybe_migrate_data(self) -> None:
         with self.rotki.data.db.conn.read_ctx() as cursor:
-            settings = self.rotki.data.db.get_settings(cursor)
-        last_migration_version = settings.last_data_migration
+            # it's important to use get_setting instead of get_settings here, because during
+            # old migrations we should not pass last_data_migration through db_settings_from_dict
+            last_migration_version = self.rotki.data.db.get_setting(cursor, 'last_data_migration')
```

Comment:
I told you to leave the last data migration in the settings. Same as version. Do not do this. There is no need to add extra logic and backwards incompatible changes here

Reply:
Actually, the comment was outdated, it's about `last_write_ts` instead of `last_data_migration`. Even if we keep `last_data_migration` in the settings table, but then call `.get_setttings`. The `last_write_ts` will be fetched from the settings table in old DB versions and passed through `db_settings_from_dict`. Which raises a warning: `Unknown DB setting last_write_ts given. Ignoring it. Should not happen so please open an issue in Github.`